data/tribes/immovables/ashes/init.lua

```diff
@@ -21,10 +23,6 @@
 --
 --    **descname**: The translatable display name. Use ``pgettext`` to fetch the string.
 --
---    **helptext_script**:  The full path to the ``helptexts.lua`` script for this immovable.
---    **NOTE: Deprecated. Immovable helptexts have been shifted to tribes initialization in the current
---    development version.**
---
```

Comment:
I decided to delete them to be consistent with buildings and workers where the passage has already been deleted. Furthermore with this change there is no instance anymore left where this is used so it might only confuse people. 

But if you insist I'll readd this but then for all entities

Reply:
There are still a couple of other references:
```
.../doc/sphinx$ grep -r helptexts.lua source/
source/lua_tribes_other.rst.org:  help of each unit's and tribe's ``helptexts.lua`` script.
source/lua_tribes_units.rst.org:``data/tribes``. Each subdirectory needs to have a ``register.lua``, an ``init.lua`` and a ``helptexts.lua``
source/lua_tribes_units.rst.org:* ``helptexts.lua`` will provide the information used by the in-game help
source/lua_tribes_workers.rst.org:**helptext_script**: The full path to the ``helptexts.lua`` script for this worker.
source/lua_tribes_workers.rst.org:Each worker has a ``helptexts.lua`` script, which is located in the same directory as its ``init.lua`` script.
source/lua_tribes_buildings.rst.org:Buildings are defined in their ``init.lua`` file. They also have a corresponding ``helptexts.lua`` file that contains their help texts.
source/lua_tribes_buildings.rst.org:    *Mandatory*. The full path to the ``helptexts.lua`` script for this building.
```

Reply:
Arrrgh.  I did only check .lua files to speed up the search I didn't think of the rst files. Thanks for pointing this out. I will clean them up as well